ambry-api/src/main/java/com.github.ambry/account/ContainerBuilder.java

```diff
@@ -162,6 +164,16 @@ public ContainerBuilder setCacheable(boolean cacheable) {
     return this;
   }
 
+  /**
+   * Sets the backup setting of the {@link Container} to build
+   * @param isBackedUp The backup setting to set.
+   * @return This builder.
+   */
+  public ContainerBuilder setIsBackedUp(boolean isBackedUp) {
```

Comment:
If you decide to change the field name, make sure to update the methods/variables in the builder too.

Reply:
👍